handbook/communication/dei.md

```diff
@@ -6,53 +6,38 @@
-*   Engage to be determined a consultant to review the entire handbook for coded language/ideas and create training materials to help us check for these ourselves
-*   Develop an apprenticeship program (this will build on the relationships we are building with code schools and organizations focused on helping folks in underserved communities break into tech)
-
-Some of the outcomes we plan to achieve during this FY are (we will add more as we solidify plans for items on the lists above):
-
-*   Every employee at the company as of 2021-12-01 will complete our standard of every employee reads and completes the exercises in Pamela Fuller’s book _The Leader’s Guide to Unconscious Bias _+ receives hard conversations training
-*   We are able to capture and regularly report on company demographics and belonging
-*   The candidate pool for every position is comprised of at least N% candidates from underrepresented groups (we are working to measure and set this goal)
-
-If at any point you have questions or feedback, you can always submit anonymously via our company meeting form (which is automatically posted to `#general` on Slack each week). We'll be creating DEI Slack channels for discussion and questions as well. If you are a candidate and have questions, you can email [hiring@sourcegraph.com](mailto:hiring@sourcegraph.com).
+Sourcegraph’s first steps towards creating a diverse team came with the decision to work remotely, and engage the right talent regardless of the country, city and timezone they were located in. As well as increasing access to talent across the globe, this decision also makes our working environment more inclusive of talent who are otherwise unable or unwilling to spend each day in an office, such as talent who take care of dependents at home, may not have access to transport, or have physical restrictions that making commuting difficult. 
+
+After deciding to work remotely, we further bolstered equity by:
+
+ - Choosing to offer compensation that was independent of location
+ - Making sure our talent was supported in working remotely - including offering home desk set-up,coworking space, and internet benefits.
+ - Working asynchronously as much as reasonable, to ensure our teammates across time zones and working schedules are included in conversations and calls for input 
+
+Our initiatives since then have been focused on identifying and acting on the potential obstacles to diversity and inclusivity in our existing teams and hiring process - the most impactful steps we could take to reduce bias and discrimination from our processes and interactions.  Up to and including FY22Q2, we:
+ - Implemented standardised, benchmarked compensation criteria related to each role
```

Comment:
An extra blank line here will make the list display correctly.
```suggestion
Our initiatives since then have been focused on identifying and acting on the potential obstacles to diversity and inclusivity in our existing teams and hiring process - the most impactful steps we could take to reduce bias and discrimination from our processes and interactions.  Up to and including FY22Q2, we:

 - Implemented standardised, benchmarked compensation criteria related to each role
```